.gitlab-ci.yml

```diff
@@ -8,21 +8,19 @@ stages:
 
 .base:
   before_script:
-    - schutzbot/ci_details.sh > ci-details-before-run
+    - schutzbot/ci_details.sh > ci-details-before-run.txt
     - cat schutzbot/team_ssh_keys.txt | tee -a ~/.ssh/authorized_keys > /dev/null
+    - mkdir /tmp/artifacts
```

Comment:
Why not create the directory first and then have the .txt, log files, etc created inside of the directory and make the artifact collection script less aware of what artifacts it needs to collect?

Reply:
I guess it was just easier to implement. Currently the CI puts pretty much everything to the current directory which is then uploaded to Gitlab CI.. So all I had to do was take any files with the relevant suffix or extension and put them in the artifacts folder.

Otherwise I think I would have to go through the tests again and make them put the logs and any important files in the artifacts directory... But I guess that makes more sense so I'll update it a bit.